docs/service-brokers/docs/040-event-activation-custom-resource.md

```diff
@@ -0,0 +1,43 @@
+---
+title: EventActivation
+type: Custom Resource
+---
+
+The `eventactivations.remoteenvironment.kyma.cx` Custom Resource Definition (CRD) is a detailed description of the kind of data and the format used to create an Event Bus Subscription. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
+
+```
+kubectl get crd eventactivations.remoteenvironment.kyma.cx -o yaml
+```
+
+## Sample custom resource
+
+This is a sample resource that allows you to consume Events sent from the service with the `ac031e8c-9aa4-4cb7-8999-0d358726ffaa` ID in a `production` Namespace.
```

Comment:
Also with that name, we have a ui-api-layer resource to fetch schema of given Event service class.

Reply:
Done.